### ISSUE TITLE
Is the Azure Storage working with .NET Core?

### ISSUE THREAD

**Issue by itsfelipesan** [aspnet/AzureIntegration#3]
I'm trying to use the Azure Starage with .NET Core 1.1, but the project is getting errors of dependencies.
I can't use the `"WindowsAzure.Storage": "7.2.1"` dependency.

That's my project.json

```
{
  "dependencies": {
    "Microsoft.AspNetCore.Mvc": "1.1.0-preview1-final",
    "Microsoft.AspNetCore.Antiforgery": "1.1.0-preview1-final",
    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-preview1-final",
    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-preview1-final",
    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.0-preview1-final",
    "Microsoft.Extensions.Configuration.FileExtensions": "1.1.0-preview1-final",
    "Microsoft.Extensions.Configuration.Json": "1.1.0-preview1-final",
    "Microsoft.Extensions.Logging": "1.1.0-preview1-final",
    "Microsoft.Extensions.Logging.Console": "1.1.0-preview1-final",
    "Microsoft.Extensions.Logging.Debug": "1.1.0-preview1-final",
    "Microsoft.AspNetCore.Hosting": "1.1.0-preview1-final",
    "Microsoft.EntityFrameworkCore": "1.1.0-preview1-final",
    "Microsoft.EntityFrameworkCore.Design": "1.1.0-preview1-final",
    "Microsoft.EntityFrameworkCore.Relational": "1.1.0-preview1-final",
    "Identity.Dapper.MySql": "0.3.0-alpha",
    "Microsoft.AspNetCore.Diagnostics": "1.1.0-preview1-final",
    "Identity.Dapper": "0.3.0-alpha",
    "MySql.Data": "7.0.5-IR21",
    "MySql.Data.EntityFrameworkCore": "7.0.5-IR21",
    "AspNet.Security.OAuth.Validation": "1.0.0-alpha3-0188",
    "Microsoft.AspNetCore.Authentication.JwtBearer": "1.1.0-preview1-final",
    "OpenIddict": "1.0.0-*",
    "Npgsql": "3.1.8",
    "Npgsql.EntityFrameworkCore.PostgreSQL": "1.0.2",
    "Identity.Dapper.PostgreSQL": "0.3.0-alpha",
    "Microsoft.AspNetCore.StaticFiles": "1.1.0-preview1-final",
    "Dreampper.Crosscutting": "1.0.0-*",
    "WindowsAzure.Storage": "7.2.1"
  },

  "tools": {
    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
      "version": "1.0.0-preview1-final",
      "imports": "portable-net45+win8+dnxcore50"
    },
    "Microsoft.EntityFrameworkCore.Tools.DotNet": "1.0.0-preview3-final"
  },

  "frameworks": {
    "netcoreapp1.1": {
      "dependencies": {
        "Microsoft.NETCore.App": {
          "type": "platform",
          "version": "1.1.0-preview1-001100-00"
        }
      },
      "imports": [
        "dotnet5.6",
        "dnxcore50"
      ]
    }
  },

  "buildOptions": {
    "emitEntryPoint": true,
    "preserveCompilationContext": true
  },

  "runtimeOptions": {
    "gcServer": true
  },

  "publishOptions": {
    "include": [
      "wwwroot",
      "Views",
      "appsettings.json",
      "web.config"
    ]
  },

  "scripts": {
    "postpublish": [ "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%" ]
  }
}

```

**Comment by GuardRex** [comment]
@itsfelipesan Try adding an `imports` for ...
```
"portable-net45+win8"
```

**Comment by adrianMaersk** [comment]
I have the same problem, I can't use WindowsAzure.Storage": "7.2.1 with .Net standart 1.6.1

**Comment by muratg** [comment]
@itsfelipesan @adrianMaersk adding imports statement as @GuardRex recommended should work this.

@bleroy - do you know when this would be usable without imports? Could you chase folks as needed? :)

**Comment by GuardRex** [comment]
IIRC this was related to OData, which still required the imports. OData is still baked into Azure Storage, but they plan to remove it at some point.

The last word from @markdstafford over at OData is https://github.com/OData/odata.net/issues/568#issuecomment-230068541

**Comment by glennc** [comment]
We will fix this in 2.0. All the dependencies have been updated, but there are breaking changes in them so we will need to wait for 2.0 to update the version we depend on.

**Comment by glennc** [comment]
Will be fixed by https://github.com/aspnet/Logging/issues/562
